vcs/vcsvtk/pipeline1d.py

```diff
@@ -103,7 +103,7 @@ def plot(self, data1, data2, tmpl, grid, transform, **kargs):
                 self._context().canvas.plot(m, donotstoredisplay=True)
 
         ren2 = self._context().createRenderer()
-        ren2.SetLayer(1)
+        self._context().setLayer(ren2, tmpl.data.priority)
```

Comment:
@danlipsa let's hold on on this, I think you want the same priority asthe other renderer which can be 0 if line width is zero. No? I'm pushing a PR with slowdown test in the second. Let's talk again after this ok?